README.md

```diff
@@ -18,6 +18,7 @@ yarn eslint - checks for eslint errors
 ```javascript
 window.OnetAdsConfig = {
     shouldShowLoader: true, // boolean
+    shouldRemoveDecimalFromPrice: false, // boolean
```

Comment:
dopiszmy to w README, proszę

Reply:
Fakt mała rozbieżność w nazwach, bo zmieniałem na samym końcu